discovery/include/aws/discovery/GGGroup.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+/* Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+
+* This file is generated
+*/
+#include <aws/discovery/GGCore.h>
+
+namespace Aws
+{
+    namespace Discovery
+    {
+        class AWS_DISCOVERY_API GGGroup final
+        {
+          public:
+            GGGroup() = default;
```

Comment:
They're GGGREAT!

I was going to suggest a namespace or the use of Greengrass, but then it turns out this is the nomenclature GG uses :\